examples/guides/messaging/messaging-with-sqs/messaging-with-sqs.md

```diff
@@ -0,0 +1,298 @@
+The high level sections of this guide are as follows:
+
+- [What you'll build](#What-youll-build)
+- [Prerequisites](#Prerequisites)
+- [Implementation](#Implementation)
+  - [Creating the project structure](#Creating-the-project-structure)
+  - [Developing the scenario](#Developing-the-scenario)
+- [Deployment](#Deployment)
+  - [Deploying locally](#Deploying-locally)
+  - [Deploying on Docker](#Deploying-on-Docker)
+
+## What you'll build
+
+Let's consider a scenario where a fire alarm send fire alerts to a Amazon SQS queue. As the message duplication is not an issue, an Amazon SQS Standard Queue is used as the queue. A fire alarm listener is polling the messages in the SQS queue. When fire alerts are available, it will consume the messages from the queue and remove the messages from the queue. Consumed messages are showed in the console to the User. 
+
+Here, the fire alarm is sending fire alerts periodically from a Ballerina worker where listener polls in another worker. Both sent messages and received messages are printed in the console. 
+
+As there can be multiple alert messages available in the queue, the listener is configured to consume more than one message at a time.
+
+The following diagram illustrates the scenario:
+
+![Message flow diagram image](resources/sqs-alert.png)
+
+
+## Prerequisites
+- [Ballerina Distribution](https://ballerina.io/learn/getting-started/)
+- A Text Editor or an IDE 
+  > **Tip**: For a better development experience, install one of the following Ballerina IDE plugins: [VSCode](https://marketplace.visualstudio.com/items?itemName=ballerina.ballerina), [IntelliJ IDEA](https://plugins.jetbrains.com/plugin/9520-ballerina)
+- [Amazon SQS Account](https://aws.amazon.com/sqs/)
+
+
+## Implementation
+> If you want to skip the basics and move directly to the [Testing](#testing) section, you can download the project from GitHub and skip the [Implementation](#implementation) instructions.
+
+Take a look at the code samples below to understand how to implement the scenario. 
+
+Following code creates a new queue in Amazon SQS with the configuration provided in a file.
+
+**create_notification_queue.bal**
+```ballerina
+import ballerina/config;
+import wso2/amazonsqs;
+
+function createNotificationQueue(string queueName) returns string|error {
+    // Amazon SQS client configuration
+    amazonsqs:Configuration configuration = {
+        accessKey: config:getAsString("ACCESS_KEY_ID"),
+        secretKey: config:getAsString("SECRET_ACCESS_KEY"),
+        region: config:getAsString("REGION"),
+        accountNumber: config:getAsString("ACCOUNT_NUMBER")
+    };
+
+    // Amazon SQS client
+    amazonsqs:Client sqsClient = new(configuration);
+
+    // Create SQS Standard Queue for notifications
+    string|error queueURL = sqsClient->createQueue(queueName, {});
+    return queueURL;
+}
+```
+
+Following code generates fire alert notifications periodically and are sent to the above created SQS queue.
+
+**notify_fire.bal**
```

Comment:
Shall we add this code segment using INCLUDE_CODE tag?
Please refer https://github.com/wso2/ballerina-integrator/tree/master/www#a-include-a-code-file

Reply:
Added INCLUDE_CODE tag and moved the code to correct guide location.